vendor/nanogui/common.cpp

```diff
@@ -92,7 +93,7 @@ void mainloop(int refresh) {
             }
 
             /* Wait for mouse/keyboard or empty refresh events */
-            int result = pollEvents ? SDL_PollEvent(&theEvent) : SDL_WaitEventTimeout(&theEvent, refresh);
+            int result = SDL_WaitEventTimeout(&theEvent, throttle);  // uses SDL_PollEvent(&theEvent) when throttle == 0
```

Comment:
I think this can just be `SDL_WaitEventTimeout`, since calling with 0 is equivalent to `PollEvent` anyway.

Reply:
It is... the other part is just in a comment so we don't forget 😀

Reply:
Ah crap, GitHub wrapped the code and I didn't notice the //